app/backend/src/couchers/servicers/communities.py

```diff
@@ -180,6 +180,43 @@ def ListAdmins(self, request, context):
                 next_page_token=str(admins[-1].id) if len(admins) > page_size else None,
             )
 
+    def AddAdmin(self, request, context):
+        with session_scope() as session:
+            user = session.execute(select(User).where_username_or_email_or_id(request.user)).scalar_one_or_none()
+            if not user:
+                context.abort(grpc.StatusCode.NOT_FOUND, errors.USER_NOT_FOUND)
```

Comment:
I think better to pass in the `user_id` and do something like

```py
session.execute(select(User).where_users_visible(context).where(User.id == request.user_id)).scalar_one_or_none()
```

(The `where_[...]_email` function is only for admins, otherwise emails are private).